src/bootstrap/src/lib.rs

```diff
@@ -958,6 +958,65 @@ impl Build {
         })
     }
 
+    fn run_tracked(&self, command: BootstrapCommand) -> CommandOutput {
```

Comment:
Note: this is just copy-paste of `run_cmd`, but adapted to returning `CommandOutput` instead of just `bool`. The duplication is removed in a later commit.